operations/rates.md

```diff
@@ -287,183 +227,210 @@ The price in the response is dependent on the enterprise's [pricing](configurati
+                "NetValue": 93.46,
+                "TaxValue": 6.54
+              }
             ]
+          }
         }
-    ],
-    "TimeUnitStartsUtc": [
-        "2022-01-01T23:00:00Z",
-        "2022-01-02T23:00:00Z",
-        "2022-01-03T23:00:00Z"
-    ]
+      ]
+    }
+  ],
+  "TimeUnitStartsUtc": [
+    "2022-01-01T23:00:00Z",
+    "2022-01-02T23:00:00Z",
+    "2022-01-03T23:00:00Z"
+  ]
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Currency` | string | required | ISO-4217 code of the [Currency](currencies.md#currency). |
 | `TimeUnitStartsUtc` | array of string | required | Set of all time units covered by the time interval; expressed in UTC timezone ISO 8601 format. |
-| `BaseAmountPrices` | array of [Amount value](accountingitems.md#amount-value) | required | Base prices of the rates for each time unit covered by the time interval. |
-| `CategoryPrices` | array of [Resource category pricing](#resource-category-pricing) | required | Resource category prices. |
-| `CategoryAdjustments` | array of [Resource category adjustment](#resource-category-adjustment) | required | Resource category adjustments. |
-| `RelativeAdjustment` | decimal | required | Specific amount which shows the difference between this rate and the base rate. |
-| `AbsoluteAdjustment` | decimal | required | Relative amount which shows the difference between this rate and the base rate. |
-| `EmptyUnitAdjustment` | decimal | required | Price adjustment for when the resource booked with this rate is not full to capacity. |
-| `ExtraUnitAdjustment` | decimal | required | Price adjustment for when the resource booked with this rate exceeds capacity. |
+| `BaseAmountPrices` | array of [Amount](_objects.md#amount) | required | Base prices of the rates for each time unit covered by the time interval. |
+| `CategoryPrices` | array of [Category pricing](rates.md#category-pricing) | required | Resource category prices. |
+| `CategoryAdjustments` | array of [Category adjustment](rates.md#category-adjustment) | required | Resource category adjustments. |
+| `AgeCategoryAdjustments` | array of [Age category adjustment](rates.md#age-category-adjustment) | required |  |
+| `RelativeAdjustment` | number | required | Specific amount which shows the difference between this rate and the base rate. |
+| `AbsoluteAdjustment` | number | required | Relative amount which shows the difference between this rate and the base rate. |
+| `EmptyUnitAdjustment` | number | required | Price adjustment for when the resource booked with this rate is not full to capacity. |
+| `ExtraUnitAdjustment` | number | required | Price adjustment for when the resource booked with this rate exceeds capacity. |
+| ~~`DatesUtc`~~ | ~~array of string~~ | ~~optional~~ | **Deprecated!** Use `TimeUnitStartsUtc` instead.|
+| ~~`BasePrices`~~ | ~~array of number~~ | ~~required~~ | **Deprecated!** Use `BaseAmountPrices` instead.|
+
+#### Category pricing
```

Comment:
suggestion: Rename to "Resource category pricing"

Reply:
fixed as a part of this [commit](https://github.com/MewsSystems/gitbook-connector-api/pull/635/commits/4ffadcbf4cb6e89054bcc8bb1c90969ee7377bad)